ui.frontend/src/view/FormFieldBase.js

```diff
@@ -184,6 +190,10 @@ export default class FormFieldBase extends FormField {
         return this.parentView.getModel().name;
     }
 
+    #getPanelTitle() {
+        return this.parentView.getModel().label ? this.parentView.getModel().label.value: "Root Panel";
```

Comment:
there is no such thing as Root Panel in the form.
we should not have have logic of getting panel title here. pls get it from event payload or introduce it at level of event payload inside guidebridge.